data/fields/gender_segregated.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key": "gender_segregated",
+    "type": "check",
+    "label": "Gender separated areas or timeslots"
```

Comment:
I thing the label needs to be shorter to look well in the UI. We will see..

Reply:
The UI should be yes/no/nil (unknown, default). 
AFAIK we can have the value `yes` an a label that is more descriptive. That could help with getting the field label shorter.

Reply:
Note to self: does the wiki talk about areas and timeslots? Do we map one sauna-room or the facility?; and what would the area be when it is a room? Or how would we map the room when the facility is meant?